app/views/pg_hero/home/index.html.erb

```diff
@@ -347,8 +347,10 @@
     <div id="migration2" style="display: none;">
       <pre>rails g migration remove_unneeded_indexes</pre>
       <p>And paste</p>
-      <pre style="overflow: scroll; white-space: pre; word-break: normal;"><% @duplicate_indexes.each do |query| %>
```

Comment:
Is it possible to only change the `remove_index` line instead of the proceeding line in all 3 instances?